kivy/properties.pyx

```diff
@@ -1432,7 +1452,8 @@ cdef class AliasProperty(Property):
             self.use_cache = 1
 
     def __read_only(self, _obj, _value):
-        raise AttributeError('property is read-only')
+        raise AttributeError('"{}.{}" property is readonly'
+                             .format(type(_obj).__name__, self._name))
```

Comment:
I feel like a linter :D sorry, but this syntax is (imo) just plainly ugly and only a backslash away from a shell syntax. If you separate the lines you can actually get even nicer errors from the traceback if anything occurs for example:

    raise AttributeError(
        '...'.format(
            type(_obj).__name__,
            self._name
        )
    )

and if you can't access `__name__` or `_name` or the format is wrong, you get the right line instead of a line with multiple accesses or calls. And it's more readable if you just scroll down the code on a phone (more space around specific objects, eyes will see the desired objects sooner).